### ISSUE TITLE
Return a promise when no callback is passed?

### ISSUE THREAD

**Issue by retorquere** [mtimofiiv/mongoose-findorcreate#3]
Would it be possible to return a promise when no callback was passed?

**Comment by mtimofiiv** [comment]
Good idea, promises are the future, so it would be good to do as you suggest. I will try to have this done in the next couple of weeks, or if you'd like to submit a PR, that would be even easier!

**Comment by retorquere** [comment]
https://github.com/mtimofiiv/mongoose-findorcreate/pull/4
